bundles/tampere/content-editor/view/SideContentEditor.js

```diff
@@ -521,32 +719,47 @@ Oskari.clazz.define('Oskari.tampere.bundle.content-editor.view.SideContentEditor
             me._geojson = null;
             jQuery.ajax({
                 type: 'POST',
-                data: {'featureData': JSON.stringify(requestData)},
+                data : {'featureData': JSON.stringify(requestData)},
                 url: url,
                 success: function (response) {
-                    if (me.operationMode === 'create') {
+                    spinner.stop();
+                    overlay.close();
+                    me.editMultipleFeaturesButton.setEnabled(false);
+                    me.editMultipleFeatures = false;
+                    if (me.operationMode === 'create' && !me.editMultipleFeatures) {
                         me.currentData.features[0][0] = response.fid;
                     }
 
                     me._clearFeaturesList();
                     var layer = me._getLayerById(me.layerId);
-                    me._highlighGeometries([], layer, true);
+                    me._removeHighligh(me.layerId);
                     wfsLayerPlugin.deleteTileCache(me.layerId, layer.getCurrentStyle().getName());
-                    var evt = Oskari.eventBuilder('AfterChangeMapLayerStyleEvent')(layer);
+                    // wfsLayerPlugin.refreshLayer(me.layerId);
+                    var evt = me.sandbox.getEventBuilder('AfterChangeMapLayerStyleEvent')(layer);
                     me.sandbox.notifyAll(evt);
                     me.sendStopDrawRequest(true);
 
                     okButton.setHandler(function () {
-                        setTimeout(function () {
-                            var visibilityRequestBuilder = Oskari.requestBuilder('MapModulePlugin.MapLayerUpdateRequest'),
+                        setTimeout(function() {
+                            var visibilityRequestBuilder = me.sandbox.getRequestBuilder('MapModulePlugin.MapLayerUpdateRequest'),
```

Comment:
You are removing the current version of getting requestBuilder and replacing it with the deprecated one?